libraries/AP_NavEKF3/AP_NavEKF3_Measurements.cpp

```diff
@@ -1010,6 +1010,7 @@ void NavEKF3_core::writeExtNavVelData(const Vector3f &vel, float err, uint32_t t
         return;
     }
 
+    ext_nav_vel_elements extNavVelNew {};
```

Comment:
Would be nice to construct this in one hit just before we pass it into the function.

So something like
```
const ext_nav_vel_elements extNavVelNew {
    vel,
    err,
    timeStamp_ms
};
```

Reply:
thank you @peterbarker this is a good idea. I will do it